Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640751): Gdk-CRITICAL **: 18:14:28.857: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641196): Gdk-CRITICAL **: 18:14:28.895: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641288): Gdk-CRITICAL **: 18:14:36.031: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641630): Gdk-CRITICAL **: 18:14:38.467: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641509): Gdk-CRITICAL **: 18:14:38.475: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642538): Gdk-CRITICAL **: 18:14:41.710: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641233): Gdk-CRITICAL **: 18:14:41.735: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641874): Gdk-CRITICAL **: 18:14:41.779: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641399): Gdk-CRITICAL **: 18:14:41.834: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642637): Gdk-CRITICAL **: 18:14:43.165: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642349): Gdk-CRITICAL **: 18:14:43.194: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640845): Gdk-CRITICAL **: 18:14:43.212: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640896): Gdk-CRITICAL **: 18:14:43.252: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642072): Gdk-CRITICAL **: 18:14:43.366: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640791): Gdk-CRITICAL **: 18:14:43.483: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640775): Gdk-CRITICAL **: 18:14:43.499: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642036): Gdk-CRITICAL **: 18:14:43.700: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642849): Gdk-CRITICAL **: 18:14:43.718: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641994): Gdk-CRITICAL **: 18:14:43.737: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641860): Gdk-CRITICAL **: 18:14:43.770: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642481): Gdk-CRITICAL **: 18:14:43.842: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642405): Gdk-CRITICAL **: 18:14:43.858: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640928): Gdk-CRITICAL **: 18:14:45.750: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641101): Gdk-CRITICAL **: 18:14:46.078: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640722): Gdk-CRITICAL **: 18:14:46.166: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641035): Gdk-CRITICAL **: 18:14:46.202: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640796): Gdk-CRITICAL **: 18:14:46.276: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641605): Gdk-CRITICAL **: 18:14:46.638: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642812): Gdk-CRITICAL **: 18:14:46.907: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640763): Gdk-CRITICAL **: 18:14:46.982: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640757): Gdk-CRITICAL **: 18:14:47.130: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641818): Gdk-CRITICAL **: 18:14:49.427: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641660): Gdk-CRITICAL **: 18:14:49.430: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642107): Gdk-CRITICAL **: 18:14:49.482: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641112): Gdk-CRITICAL **: 18:14:49.635: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642864): Gdk-CRITICAL **: 18:14:49.667: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642371): Gdk-CRITICAL **: 18:14:50.171: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642823): Gdk-CRITICAL **: 18:14:50.775: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642501): Gdk-CRITICAL **: 18:14:50.915: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641430): Gdk-CRITICAL **: 18:14:51.146: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640781): Gdk-CRITICAL **: 18:14:51.706: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641844): Gdk-CRITICAL **: 18:14:51.743: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642796): Gdk-CRITICAL **: 18:14:51.907: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641753): Gdk-CRITICAL **: 18:14:52.174: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641426): Gdk-CRITICAL **: 18:14:52.746: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641656): Gdk-CRITICAL **: 18:14:53.071: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642816): Gdk-CRITICAL **: 18:14:54.808: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642386): Gdk-CRITICAL **: 18:14:55.243: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641173): Gdk-CRITICAL **: 18:14:55.267: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640841): Gdk-CRITICAL **: 18:14:55.323: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640737): Gdk-CRITICAL **: 18:14:55.480: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641566): Gdk-CRITICAL **: 18:14:55.558: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642653): Gdk-CRITICAL **: 18:14:55.668: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641447): Gdk-CRITICAL **: 18:14:55.677: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640935): Gdk-CRITICAL **: 18:14:57.121: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642327): Gdk-CRITICAL **: 18:14:57.978: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642780): Gdk-CRITICAL **: 18:14:58.732: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642289): Gdk-CRITICAL **: 18:14:58.853: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642253): Gdk-CRITICAL **: 18:14:59.618: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641892): Gdk-CRITICAL **: 18:14:59.771: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642416): Gdk-CRITICAL **: 18:15:00.781: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642513): Gdk-CRITICAL **: 18:15:00.807: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642092): Gdk-CRITICAL **: 18:15:01.311: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642215): Gdk-CRITICAL **: 18:15:01.530: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642110): Gdk-CRITICAL **: 18:15:01.692: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642872): Gdk-CRITICAL **: 18:15:01.722: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640711): Gdk-CRITICAL **: 18:15:01.895: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642130): Gdk-CRITICAL **: 18:15:02.203: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642874): Gdk-CRITICAL **: 18:15:02.940: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642270): Gdk-CRITICAL **: 18:15:03.447: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641735): Gdk-CRITICAL **: 18:15:03.520: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641332): Gdk-CRITICAL **: 18:15:04.571: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642057): Gdk-CRITICAL **: 18:15:04.871: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641066): Gdk-CRITICAL **: 18:15:05.071: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641003): Gdk-CRITICAL **: 18:15:05.186: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642631): Gdk-CRITICAL **: 18:15:05.851: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641686): Gdk-CRITICAL **: 18:15:06.254: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1640825): Gdk-CRITICAL **: 18:15:06.362: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642828): Gdk-CRITICAL **: 18:15:06.954: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641683): Gdk-CRITICAL **: 18:15:07.207: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641140): Gdk-CRITICAL **: 18:15:07.542: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641834): Gdk-CRITICAL **: 18:15:07.670: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641142): Gdk-CRITICAL **: 18:15:07.674: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641704): Gdk-CRITICAL **: 18:15:07.770: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641316): Gdk-CRITICAL **: 18:15:07.818: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642199): Gdk-CRITICAL **: 18:15:07.839: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641527): Gdk-CRITICAL **: 18:15:07.907: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641967): Gdk-CRITICAL **: 18:15:07.927: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642858): Gdk-CRITICAL **: 18:15:07.931: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641080): Gdk-CRITICAL **: 18:15:08.149: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642724): Gdk-CRITICAL **: 18:15:08.182: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641807): Gdk-CRITICAL **: 18:15:08.245: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641164): Gdk-CRITICAL **: 18:15:08.265: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642459): Gdk-CRITICAL **: 18:15:08.285: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642137): Gdk-CRITICAL **: 18:15:08.363: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641470): Gdk-CRITICAL **: 18:15:08.467: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642700): Gdk-CRITICAL **: 18:15:09.039: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642310): Gdk-CRITICAL **: 18:15:09.063: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1641865): Gdk-CRITICAL **: 18:15:09.071: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1642844): Gdk-CRITICAL **: 18:15:09.351: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
...........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
[1695 9137]
...........................................................................................
[1706 9155]
..
[1723 9121]
...........
[8786 1719]
.
[9288 1250]
..
[1742 9134]
......
[9451 1102]
..
[1689 9140]
......
[1708 9113]
...
[9472 1067]
.
[1721 9158]
.......
[1713 9132]
.
[1787 9072]
..
[1752 9114]
.
[9435 1107]
.
[1863 8996]
.
[1712 9141]
..
[1664 9182]
.
[1693 9158]
..
[1715 9109]
.
[1805 9053]
.
[1898 8936]
.
[1916 8900]
..
[1688 9115]
.
[1692 9172]
.
[1790 9019]
.
[9390 1118]
.......
[1752 9164]
..
[1696 9152]
..
[2687 8026]
..
[4111 6503]
.
[9275 1254]
.
[9400 1129]
.
[2052 8733]
.
[1992 8776]
.
[2407 8346]
.
[9418 1117]
....
[2259 8543]
..
[9279 1230]
..
[9410 1134]
..
[2274 8476]
...
[9375 1196]
....
[9147 1353]
.
[1992 8780]
..
[9252 1259]
.
[2268 8481]
...
[8912 1549]
.
[9070 1441]
.
[2581 8134]
.
[9349 1216]
..
[2587 8101]
.
[9194 1334]
.
[2496 8289]
..
[8817 1671]
.
[2868 7832]
.
[8128 2371]
..
[8547 1947]
...
[8836 1676]
.
[9386 1141]
.
[3166 7506]
...
[8277 2230]
.
[8408 2088]
..
[3685 6956]
...
[8947 1584]
..
[9240 1311]
.......
[3376 7277]
...
[8167 2345]
....
[8053 2415]
..
[3764 6796]
...
[8646 1871]
....
[7773 2707]
.....
[3694 6915]
..
[4516 6063]
..
[6115 4433]
.
[8511 1941]
..
[7623 2832]
.
[4683 5865]
......
[7530 2928]
...........
[5409 5124]
.
[7124 3365]
.
[6920 3594]
...
[4193 6366]
...
[4728 5827]
...
[7482 3055]
.
[5248 5240]
.
[5357 5137]
...
[7452 3022]
.
[7551 2951]
......
[5399 5153]
...
[6656 3844]
...
[5845 4647]
...........
[5219 5300]
..
[6687 3827]
....................................................................................................................................................................................................................................................
Data Dump...
...................................
Data Dump...
.......
Data Dump...
........................................................................................
[ 162 2815]
...............................................................................................................................
[3791    5]
.....................................................
[3857   13]
......................................
[ 137 3206]
...
[ 106 3494]
..
[3743   11]
.
[3671   17]
.
[3390   28]
.......
[ 116 3154]
..
[ 184 2723]
..................
[3626   24]
......
[3100   35]
......
[ 130 3014]
....
[ 123 3013]
......
[3575   14]
.
[ 168 2855]
..
[ 194 2824]
.....
[ 141 2930]
..................
[3431   30]
.........
[3408   19]
.
[ 188 2727]
..
[ 206 2708]
.........
[ 174 2769]
.
[ 233 2642]
....
[3347   21]
....
[3436   22]
....
[ 291 2500]
.........
[3447   20]
..
[3236   25]
...
[ 293 2469]
.....
[ 232 2667]
...........
[3298   21]
....
[ 318 2342]
...
[3183   17]
........
[3086   37]
...
[3038   37]
...
[ 369 2262]
...
[ 386 2167]
...........
[3291   19]
.
[ 428 2179]
.........
[3022   54]
..
[ 414 2109]
........
[3050   45]
..
[2797   87]
....
[ 506 1993]
......
[2856   79]
.
[2941   71]
...
[ 533 2026]
.
[3086   34]
.....................
[ 631 1847]
..
[ 674 1720]
..
[1422  912]
.....
[ 834 1496]
..
[ 509 1964]
...
[2784   97]
.....
[ 619 1793]
......
[2913   56]
....
[2789   80]
...
[ 651 1794]
.........
[2865   55]
.
[2697  107]
.
[ 776 1535]
..............
[ 827 1583]
.......
[ 739 1686]
...
[2613  137]
.......
[ 921 1418]
........
[2153  349]
..
[2590  121]
..........
[36 39]
...
[1098 1270]
..........
[1250 1075]
........
[2452  185]
.....
[2559  147]
...
[2414  224]
..
[2653  125]
........
[1064 1203]
....
[2631  127]
...........
Data Dump...
............
[1575  762]
..........
[1300  976]
....
[2312  268]
..
[1639  762]
.
[1841  531]
.......
[1962  482]
...........
[1903  503]
.....
[1259 1038]
.
[2024  392]
.....
[2330  246]
..................
[1683  709]
...................
[1892  522]
....
[211   2]
......
[1464  822]
...................
[1870  521]
.
Data Dump...
..............................................
[207   9]
................
Data Dump...
...............................................................
[218   6]
..............
Data Dump...
......
[227  11]
..........................
Data Dump...
..................
[179  12]
...................................
[43 60]
.
[193  10]
.............
[38 51]
.........
Data Dump...
....
[41 70]
....
Data Dump...

[153   7]
..
Data Dump...
..
[39 43]
......
[166   7]
.......................
Data Dump...
......
[45 46]
.
Data Dump...
.....
[35 47]
................
[184  16]
...
Data Dump...
...........
[35 46]
.....
Data Dump...

Data Dump...
..
[35 50]
..
[172   4]
...
[180   7]
....
Data Dump...
.....
[49 40]
...
Data Dump...
.........
[40 50]
........
[178   9]
.
Data Dump...
...............
[181   8]
..
Data Dump...
..
Data Dump...
.......
Data Dump...
...
[53 31]
.
[42 41]
.
Data Dump...
.....
[41 40]
..........
[173   7]
.
Data Dump...
..
Data Dump...
.........
[165  11]
..............
[52 44]
..
Data Dump...
.
Data Dump...
.........
[53 42]
..
[159  10]
.
Data Dump...
.
[142   9]
.......
[49 41]
....
Data Dump...
.....
[163   5]
.
[164   8]
....
Data Dump...

Data Dump...
...
[168   6]
.......
[54 37]
......
[48 41]
......
Data Dump...
...
[161   8]
..........
[55 35]
..
[58 43]
.....
Data Dump...
....
Data Dump...
......
[148   8]
.
Data Dump...
...
Data Dump...
..
[102  22]
.
Data Dump...
.....
Data Dump...

Data Dump...
..
[133   9]
...
[177   8]
........
[55 20]
...
Data Dump...

[135  14]
...
[139  18]
..
[57 35]
.....
[60 29]
...
[138  13]
....
Data Dump...

Data Dump...
......
[60 34]
..
Data Dump...
....
[73 20]
.......
[133   7]
..
Data Dump...

[74 25]
...
Data Dump...
.
[80 21]
...
[129  12]
.....
[82 30]
.
Data Dump...
.
[88 24]
.....
[135   9]
......
Data Dump...
...
[143  14]
..
[75 34]
.
[65 24]
.......
Data Dump...
...
Data Dump...
......
[157  13]
..
[112  15]
.....
Data Dump...

Data Dump...
.
Data Dump...
....
Data Dump...

Data Dump...
.
Data Dump...
..
[86 24]
.....
[71 25]
...
Data Dump...
.....
[71 15]
......
Data Dump...
.
Data Dump...
.
[131  19]
......
[77 32]
.
[148  10]
.
Data Dump...

Data Dump...
...
Data Dump...

Data Dump...
...
Data Dump...

Data Dump...
......
Data Dump...

Data Dump...

Data Dump...
...
[118  10]
.
[150  11]
.
Data Dump...
.....
[140  13]
.
[144  11]
.
[78 25]
.......
Data Dump...
..
[107  21]
..
[92 24]
.
[109  17]
...
[119  14]
.
[82 16]
.
Data Dump...

Data Dump...
......
[70 16]
.
Data Dump...

[108  15]
.
Data Dump...

Data Dump...

[104  14]
..
Data Dump...
.
[70 24]
.
[123  18]
...........
[100  23]
.....
Data Dump...

Data Dump...

Data Dump...
...
Data Dump...
.
[108  12]
...
Data Dump...
.
[110  10]
...
Data Dump...
.
Data Dump...

Data Dump...

Data Dump...

Data Dump...
....
Data Dump...
..
Data Dump...

Data Dump...
.
Data Dump...

Data Dump...
.
Data Dump...
..
Data Dump...
.
Data Dump...
Low 2 1
Done
